domain/robstoll-lib/atrium-domain-robstoll-lib-jvm/src/main/kotlin/ch/tutteli/atrium/domain/robstoll/lib/creating/localDateAssertions.kt

```diff
@@ -13,7 +13,9 @@ fun <T: LocalDate> _year(assertionContainer: Expect<T>): ExtractedFeaturePostSte
 fun <T: LocalDate> _month(assertionContainer: Expect<T>): ExtractedFeaturePostStep<T, Int> =
     ExpectImpl.feature.manualFeature(assertionContainer, DescriptionDateTimeLikeAssertion.MONTH) { monthValue }
 
-fun <T: LocalDate> _dayOfWeek(assertionContainer: Expect<T>): ExtractedFeaturePostStep<T, DayOfWeek> =
-    ExpectImpl.feature.manualFeature(assertionContainer, DescriptionDateTimeLikeAssertion.DAY_OF_WEEK) { dayOfWeek }
 
+fun <T : Collection<*>> _day(assertionContainer: Expect<T>): ExtractedFeaturePostStep<T, Int> =
+    ExpectImpl.feature.manualFeature(assertionContainer, DescriptionDateTimeLikeAssertion.DAY) { LocalDate.now().dayOfYear }
```

Comment:
That's not the idea, you wan't to extrac the `dayOfMonth` (see description of the issue) and not from `LocalDate.now()` but from the subject of the assertion. Note, the lambda you are in is `T.() -> Unit` where `T` is the type of the subject